### PR TITLE
Add new fields required by custom apps

### PR DESCRIPTION
**Pull request by olaughter**
# Description

Please include:

- a summary of the changes
- links to any related issue/ticket
- any additional relevant motivation and context
- details of any dependency updates that are required for this change

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [ ] Skip auto-tagging
- [x] Patch
- [ ] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail (integrated soon)
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Before submitting

- [ ] I've read and followed all steps in the [Making a pull request](https://github.com/climatepolicyradar/cpr-sdk/blob/main/.github/CONTRIBUTING.md#making-a-pull-request)
    section of the `CONTRIBUTING` docs.
- [ ] I've updated or added any relevant docstrings following the syntax described in the
    [Writing docstrings](https://github.com/climatepolicyradar/cpr-sdk/blob/main/.github/CONTRIBUTING.md#writing-docstrings) section of the `CONTRIBUTING` docs.
- [ ] If this PR fixes a bug, I've added a test that will fail without my fix.
- [ ] If this PR adds a new feature, I've added tests that sufficiently cover my new functionality.

### CODE REVIEW

**Review by jesse-c — COMMENTED** [review]
Should not these new fields be `Optional`, so the model is backwards-compatible?

**Comment by olaughter** [comment]
> Should not these new fields be Optional, so the model is backwards-compatible?

Agree having worked through the two options in slack this is the better as it unblocks other changes. We've also said it would be good to come back and make them required later as it will provide some checks confirming the data is there

File: src/cpr_sdk/version.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 _MAJOR = "1"
-_MINOR = "1"
+_MINOR = "3"
```
Comment (jesse-c) [inline src/cpr_sdk/version.py:2]: Why was 2 skipped?

File: src/cpr_sdk/pipeline_general_models.py [hunk 2]
```diff
@@ -31,11 +32,15 @@ class BackendDocument(BaseModel):
...
-
+    corpus_import_id: Optional[str] = ""
+    corpus_type_name: Optional[str] = ""
+    collection_title: Optional[str] = ""
+    collection_summary: Optional[str] = ""
     type: str
     source: str
     category: str
     geography: str
+    geographies: list[str] = []
```
Comment (jesse-c) [inline src/cpr_sdk/pipeline_general_models.py]: **thought:** Is there a difference between an empty list and an optional list (which may be empty)? 🤔 (This is just a thought to be ignored)

**Review by jesse-c — APPROVED** [review]
LGTM

File: src/cpr_sdk/version.py (hunk 1 above)
Comment (olaughter) [reply src/cpr_sdk/version.py:2]: Only because I used it on this one: https://github.com/climatepolicyradar/cpr-sdk/pull/76/files

Comment (jesse-c) [reply src/cpr_sdk/version.py:2]: haha, of course, sorry!

File: src/cpr_sdk/pipeline_general_models.py (hunk 2 above)
Comment (olaughter) [reply src/cpr_sdk/pipeline_general_models.py]: Thinking about it, maybe this is the approach we want above too... It basically sets `[]` as a default value if no other value is provided. But without allowing the value to be `None`.

Comment (jesse-c) [reply src/cpr_sdk/pipeline_general_models.py]: The general data modelling approach I take is that `None` of something is different to the empty value (`""` or `[]`) of something, and thus prefer `Optional`s over potentially misleading empty values. For example, is the value missing or empty? With only a list, we'll never know.

I don't have a strong opinion on it for this data modelling decision though, so happy with either approach.

Comment (olaughter) [reply src/cpr_sdk/pipeline_general_models.py]: Taken this approach, thanks! 👍

**Pull request merged**